csharp/ql/src/Diagnostics/CompilerError.ql

```diff
@@ -0,0 +1,16 @@
+/**
+ * @name Compilation error
+ * @description A compilation error can cause extraction problems, and could lead to inaccurate results.
+ * @kind problem
+ * @problem.severity recommendation
+ * @precision medium
```

Comment:
I guess we would like to have this query run by default (but not shown)? Changning precision to `high` would do that.